tracing-appender/src/lib.rs

```diff
@@ -1,15 +1,80 @@
+//! Writers for logging events/spans
```

Comment:
```suggestion
//! Writers for logging events and spans
```

Reply:
Also, can we add this top-level crate documentation to the crate's README as well?

Reply:
Yes, I just forgot to include a README for this crate, I'll create one now.